web/controllers/redcap/http_client.ex

```diff
@@ -12,9 +12,26 @@ defmodule ResearchResource.Redcap do
     filter_fields(res, instrument)
   end
 
+  def save_record(data) do
+    # add complete status to the record
+    record = Map.merge(data, %{user_details_complete: 2, consent_complete: 2})
+
+    {:ok, payload} = Poison.encode(record)
+    body = [
+      token: @redcap_token,
+      content: "record",
+      format: "json",
+      type: "flat",
+      overwriteBehavior: "normal",
+      data: "[#{payload}]"
+    ]
+
+    res = HTTPoison.post(@redcap_url, {:form, body}, [])
+  end
+
   defp filter_fields({:ok, res}, instrument) do
     {:ok, data} = Poison.Parser.parse(res.body)
 
     consent_questions = Enum.filter(data, fn(e) -> e["form_name"] == instrument end)
```

Comment:
Would you mind naming `e` a little better, would `question` be what this argument could be?